rosdep/python.yaml

```diff
@@ -8294,6 +8294,9 @@ python3-scp:
   debian: [python3-scp]
   fedora: [python3-scp]
   opensuse: [python3-scp]
+  rhel:
+    '*': [python3-scp]
```

Comment:
@cottsay is it [`python3-scp`](https://centos.pkgs.org/8/epel-x86_64/python3-scp-0.14.1-1.el8.noarch.rpm.html) or [`python-scp`](https://src.fedoraproject.org/rpms/python-scp)?

Reply:
Actually, this touches on the reason why the Fedora dashboard lists EPEL 7 even though I said it wasn't available: the source package hasn't been re-built to produce a `python3-` subpackage and only produces a `python2-` subpackage.

Reply:
Thanks for the clarification @cottsay.